ci/bot-production-notify-1-day.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: python
+    tag: 3.8
+
+inputs:
+- name: src-repository
+
+run:
+  path: bash
+  args:
+  - -c
+  - |
+    cd src-repository
+    pip install --upgrade pip
+    pip install -r requirements.txt
+    # Do not run yet python ./cli notify-1-day
```

Comment:
Should this be uncommented now if this is ready to run?

Reply:
Yup... Need to add the daily trigger too.